geode-core/src/main/java/org/apache/geode/distributed/internal/membership/NetView.java

```diff
@@ -597,6 +610,9 @@ public void toData(DataOutput out) throws IOException {
     DataSerializer.writeIntArray(failureDetectionPorts, out);
     // TODO expensive serialization
     DataSerializer.writeHashMap(publicKeys, out);
+    if (InternalDataSerializer.getVersionForDataStream(out).compareTo(Version.GEODE_120) >= 0) {
```

Comment:
geode 1.3.0 has been released so this and other places need to be changed to GEODE_140